### ISSUE TITLE
Player stops after a minute

### ISSUE THREAD

**Issue by Roxxers** [twireapp/Twire#192]
Have had this bug since I downloaded the app. Playing a stream works for about one minute before the player just stops. If this is pip then the player just disappears. If I am in the app it will stop and return to the my streams page. 

Version: 2.9.2
OS: Graphene OS Latest build

**Comment by Roxxers** [comment]
Update on this, it seems to be chat related. The number of chat messages seems related to how quickly the stream will crash. Chill streams with low viewers sometimes never crash at all. While ESL csgo streams with constant chat spam crash within seconds.

**Comment by GitJamz** [comment]
I have the exact same issue and I have had since the first day I tried Twire, like 8 months ago. Now on 3 different installs and phones with different OS's.
I Too watch CS go streams with a lot and lot of chats. And it happens everytime.

How I bypass it is I use third party player - kodi - everytime. It really sucks though and Ive been wanting to report it for a long time but lazyness...

**Comment by Roxxers** [comment]
> How I bypass it is I use third party player

Yes I do the same thing and the app works just fine with this. I will say for me it is hit and miss, I have had some streams work and not crash but its too random for me to really understand why. But using external player works and does not crash the app.

**Comment by GitJamz** [comment]
@samfundev got what you need to escalate this further? I would guess based on above speculations it should be quite reproduceable.

**Comment by GitJamz** [comment]
Anyone working on this?  If we identified the problem correct and devs can replicate this, can we just have a option to disable chat completly.
Would solve our issue short term at least.

**Comment by samfundev** [comment]
Unfortunately, I am not able to reproduce this issue. I opened some top streams that had a lot of viewers and couldn't get Twire to crash after watching for a minute. I don't know if both of you are using GrapheneOS but I am testing this on Android 11. If either of you could get a log from when the app crashed, perhaps I could look into the issue further. I could also create a build for you that prevents Twire from connecting to chat but ideally, I'd like to fix the crash issue instead of adding an option to disable chat.

**Comment by GitJamz** [comment]
> Unfortunately, I am not able to reproduce this issue. I opened some top streams that had a lot of viewers and couldn't get Twire to crash after watching for a minute. I don't know if both of you are using GrapheneOS but I am testing this on Android 11. If either of you could get a log from when the app crashed, perhaps I could look into the issue further. I could also create a build for you that prevents Twire from connecting to chat but ideally, I'd like to fix the crash issue instead of adding an option to disable chat.

I dont use GrapheneOS. I use CalyxOS at the moment but I had the exact same problem on LineageOS 16 and LineageOS 17.1.

Have you tried watching the stream for a little longer? Like 3-4 minutes? I have the feeling that first time it can sometimes work for like 4-5 minutes but then consequent tries after that can crash within seconds.

Please describe for a newbie how to create the log you need and I will get it for you! Thank you for helping

**Comment by samfundev** [comment]
> I dont use GrapheneOS. I use CalyxOS at the moment but I had the exact same problem on LineageOS 16 and LineageOS 17.1.

Interesting, I'm using stock Android but I wonder if this is an issue with using other OSes. I'll see if I can find a way to emulate one of those OSes.

> Have you tried watching the stream for a little longer? Like 3-4 minutes? I have the feeling that first time it can sometimes work for like 4-5 minutes but then consequent tries after that can crash within seconds.

I've tried watching a stream for 10 minutes and it hasn't happened.

> Please describe for a newbie how to create the log you need and I will get it for you! Thank you for helping

You can use [this app](https://f-droid.org/en/packages/com.tananaev.logcat/) on F-Droid to get a log. It should describe the general steps that you need to take to get it working.
